userspace/libscap/userspace_flag_helpers.h

```diff
@@ -2,16 +2,24 @@
 
 #include <assert.h>
 #include <fcntl.h>
+#if !defined(_WIN32)
```

Comment:
Is there a more compact way of grouping these includes depending on the platform?